Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -110,19 +105,17 @@ enum class HomeSections(
 
 @Composable
 fun JetsnackBottomBar(
-    navController: NavController,
+    shouldBeShown: Boolean,
```

Comment:
Just trying to get my bearings for idiomatic Compose code: what you think about avoiding this parameter, and instead just wrapping the `JetsnackBottomBar` in an `if`? To me that feels a little bit closer to the "if you don't want to show a composable, just don't call it" idea.

Reply:
Yeah, good point! Fixed, thank you